apps/transport/lib/transport_web/templates/aoms/index.html.eex

```diff
@@ -1,17 +1,17 @@
 <h2>Situation des AOM pour l’ouverture des données transport en commun</h2>
-
-<table class="table">
+<p><%= link "Télécharger en CSV", to: aoms_path(@conn, :csv) %></p>
```

Comment:
```suggestion
<div class="is-centered">
  <a class="button" href="<%= aoms_path(@conn, :csv) %>" role="link">
      <i class="icon icon--download" aria-hidden="true"></i>
      Télécharger en CSV
  </a>
</div>
```

to get 
![image](https://user-images.githubusercontent.com/3987698/71247942-abaed780-2311-11ea-9ba0-d7f07f40d988.png)